agent/agentlocal/agent_local.go

```diff
@@ -192,7 +195,7 @@ func (s *Server) Reload(ctx context.Context, req *agentlocalpb.ReloadRequest) (*
 // runGRPCServer runs gRPC server until context is canceled, then gracefully stops it.
 func (s *Server) runGRPCServer(ctx context.Context, listener net.Listener) {
 	l := s.l.WithField("component", "local-server/gRPC")
-	l.Debugf("Starting gRPC server on http://%s/ ...", listener.Addr().String())
+	l.Debugf("Starting gRPC server on unix:%s ...", listener.Addr().String())
```

Comment:
existing pmm-agents should keep working via `http://`

Reply:
only new pmm-agents should use sockets

Reply:
that can be checked by the existence of the port configuration in `pmm-agent.yaml` file

Reply:
I understand the grpc server is used internally only.
Originally it listened on a random port.

Since it's for internal purpose only, I believe we don't need to keep this backwards compatible.
Any thoughts?

Reply:
if pmm-admin doesn't connect to pmm-agent via gRPC maybe it's better to completely remove support of gRPC?